WeScan/Session/CaptureSession+Orientation.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  CaptureSession+Orientation.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 23/11/2018.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import CoreMotion
+import Foundation
+
+/// Extension to CaptureSession with support for automatically detecting the current orientation via CoreMotion
+/// Which works even if the user has enabled portrait lock.
+extension CaptureSession {
+    /// Detect the current orientation of the device with CoreMotion and use it to set the `editImageOrientation`.
+    func setImageOrientation() {
+        var motion: CMMotionManager!
```

Comment:
Can't you do the initialization in 1 line instead of 2?
`var motion = CMMotionManager()`?

Reply:
👍